packages/astroport/src/asset.rs

```diff
@@ -245,9 +245,9 @@ impl AssetInfo {
                 addr_validate_to_lower(api, contract_addr.as_str())?;
             }
             AssetInfo::NativeToken { denom } => {
-                if denom != &denom.to_lowercase() {
+                if !denom.contains('/') && denom != &denom.to_lowercase() {
```

Comment:
Maybe better to check only ibc denoms here? Like `!denom.start_with("ibc/")`

Reply:
Yes, good advice! Thanks